deploy/ansible/roles/legion_enclave/templates/airflow-values.yaml.j2

```diff
@@ -28,11 +28,14 @@ postgres:
   port: 5432
 
 core:
-  logging_level: info
+  logging_level: debug
```

Comment:
Maybe we should disable debbuging back?

Reply:
don't thinks we really suffer from debug, the output is not so verbose but it helps to investigate some issues on the flight without the need to reconfigure pods and redeploy.